content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -245,7 +245,7 @@ Existing static backends that you configure with runtime flags are not affected
 
 The AuditSink policy differs from the legacy audit runtime policy. This is because the API object serves different use cases. The policy will continue to evolve to serve more use cases.
 
-The `level` field applies the given audit level to all requests. The `stages` field is now a whitelist of stages to record.
+The `level` field applies the given audit level to all requests. The `stages` field is now a allowlist of stages to record.
```

Comment:
the entire `Dynamic backend` section can be removed in 1.19, per https://github.com/kubernetes/kubernetes/pull/91502

Reply:
I'm not sure what to do here so I'm just going to 'resolve' Seems like removing dynamic backend should be in a different PR.

Reply:
I don't think this reads very clearly, and the section should be removed. I'd probably drop this change.